src/NoUnnecessaryTrailingUnderscore.elm

```diff
@@ -680,7 +670,7 @@ report patternsAndBody context =
                                 |> List.map (\var -> ( var.name, [] ))
                                 |> Dict.fromList
                     in
-                    { errors = List.filterMap (error expression (addNewScope names context.scopes)) declaredVariables
+                    { errors = List.filterMap (error expression Dict.empty (addNewScope names context.scopes)) declaredVariables
```

Comment:
Is it correct to pass `Dict.empty` here?

Reply:
I think it is. This didn't cause any issues on a large work project of mine at least